reporting-bugs/readme.md

```diff
@@ -0,0 +1,263 @@
+
+<!--{{<note type="activity" title="Step by step improvement, 5m" >}}-->
+
+1. Set a [timer for 5 minutes](https://www.google.com/search?q=timer+for+5+minutes).
+1. Popcorn around the room and name three problems with this bug report. How should it be improved?
+
+Send it back!
+
+<!--{{</note>}}-->
+
+### 🐛 Bug Report Improvement 2
+
+<details>
+<summary>Bug: cannot create a new file on the command line</summary>
+
+#### Steps to reproduce:
+
+1. Open [repo](/link-to-repo) in VSCode
+1. Open terminal and type `touch newfile.js`
+1. Get error `'touch' is not recognized as an internal or external command, operable program or batch file.`
+
+#### Expected behaviour
+
+- I expected to create a new file in the terminal using the `touch` command.
+
+#### Actual behaviour
+
+- Instead, I got an error message `'touch' is not recognized as an internal or external command, operable program or batch file.`
+</details>
+<!--{{<note type="activity" title="Precision and clarity, 5m" >}}-->
+
+Our bug reporter has worked hard on improving their bug report. But there is one crucial detail missing.
+
+1. Set a [timer for 5 minutes](https://www.google.com/search?q=timer+for+5+minutes).
+1. Popcorn around the room and guess/suggest how this bug report could be improved with one more detail.
+
+<!--{{</note>}}-->
+
+## 🐛 Bug Report Final Form
+
+See how much easier is to fix in the final form? What is the bug in this report?
+
+<details>
+<summary>Bug: cannot create a new js file on the command line</summary>
+
+#### Steps to reproduce:
+
+1. Open [repo](/link-to-repo) in VSCode
+1. Open terminal and type `touch newfile.js`
+1. Get error `'touch' is not recognized as an internal or external command, operable program or batch file.`
+
+#### Expected behaviour
+
+- I expected to create a new file in the terminal using the `touch` command.
+
+#### Actual behaviour
+
+- Instead, I got an error message `'touch' is not recognized as an internal or external command, operable program or batch file.`
+
+#### Environment details:
+
+VSCode on Windows 10, Node v14.15.4, NPM v6.14.10, React v17.0.1, deployed to Netlify.
+
+</details>
+
+## Wrapping up
+
+Recapping the key points:
+
+Include as much context as you can, including links, line numbers, sample code, etc. Use code blocks to mark up your code. If you use screenshots, they should only show the _visual render_ (**never** screenshot code - paste code as text).
+
+### 🧑🏾‍💻🙋 Bug reports contain
+
+1. 🔗 Links 👍
+1. 🎯 Objectives - what are you actually trying to do? 👍
+1. 🖼️ Screenshots of UI if relevant 👍
+1. 📝 `Code blocks` 👍
+1. 🪸 Environment details 👍
+1. 📷 ~~Screenshots of code~~ 🙅
+
+## 📚 Further reading
+
+- [How to ask](https://stackoverflow.com/help/how-to-ask)
+- [The programming duck](https://rubberduckdebugging.com/)
+- [Asking Questions](https://curriculum.codeyourfuture.io/guides/asking-questions/)
+- [Reading Error Messages](https://www.codementor.io/@erikpukinskis/how-to-read-an-error-message-6sebm8ik3)
```

Comment:
There's also:

- https://codeblog.jonskeet.uk/2010/08/29/writing-the-perfect-question/ - focused on Stack Overflow, by a very experienced contributor
- http://catb.org/esr/faqs/smart-questions.html - pretty blunt and a old-fashioned at this point, I think we'd have to include that caveat if pointing to it, but I think still potentially useful